Makd.mak

```diff
@@ -36,6 +36,15 @@ F ?= devel
 # use "." for the current directory)
 SRC = src
 
+# Directory where all the integration tests are
+ifndef INTEGRATIONTEST
```

Comment:
Right! Currently it is a bit messy what can be overridden by `Config.mak`, I should probably clean that up at some point.